digits/pretrained_model/tasks/torch_upload.py

```diff
@@ -30,6 +41,30 @@ def get_weights_path(self):
         return os.path.join(self.job_dir, "_Model.t7")
 
     @override
+    def get_deploy_path(self):
+        """
+        Get path to file containing model def for deploy/visualization
+        """
+        return os.path.join(self.job_dir,"original.lua")
```

Comment:
@TimZaman This PR is just of my top of branch before finishing my internship. Its the same as those two combines, but rebased with master, and Gradient Ascent for Torch.